packages/web/package.json

```diff
@@ -57,7 +57,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "yarn ts:check && yarn test --watchAll"
+      "pre-push": "yarn ts:check && yarn test --watchAll=false"
```

Comment:
Can you just omit the watchAll flag?
```
"pre-push": "yarn ts:check && yarn test"
```

Reply:
No. I can't because in the watch mode the task doesn't return code as a result of running `yarn test` that we need to continue to push